src/views/ConfirmEmail.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { KYCLink } from '../components/PredefinedLinks';
+
+const ConfirmEmailComponent = ({ confirmationId }) => (<div>
+  <h2>Confirmation of e-mail and second factor auth setup page</h2>
+  <div>Got confirmation id: {confirmationId}</div>
+  <KYCLink>Registration succeed go to KYC page</KYCLink> <br />
```

Comment:
Use `<p>` around text instead of using `div` or `br`. The `<br/>` tag should never be used.